aea/helpers/exec_timeout.py

```diff
@@ -273,4 +271,4 @@ def _remove_timeout_watch(self) -> None:
         """
         if self._future_guard_task and not self._future_guard_task.done():
             self._future_guard_task.cancel()
-            self._future_guard_task = None
+        self._future_guard_task = None
```

Comment:
What is the justification for this change?

Reply:
https://github.com/valory-xyz/open-aea/pull/413#discussion_r1008559905